measures/bestest_building_thermal_envelope_and_fabric_load/measure.rb

```diff
@@ -378,26 +396,49 @@ def run(model, runner, user_arguments)
     model.getBuilding.setName("BESTEST Case #{case_num}")
     runner.registerInfo("Renaming Building > #{model.getBuilding.name}")
 
-
     # set timesteps per hour
     timestep = model.getTimestep
-    timestep.setNumberOfTimestepsPerHour(4)
+    timestep.setNumberOfTimestepsPerHour(6)
 
     # set shadow calcs
     shadow_calc_freq = model.getShadowCalculation
     shadow_calc_freq.setShadingCalculationUpdateFrequency(1)
 
     # set ground temps
-    ground_temps = model.getSiteGroundTemperatureBuildingSurface
-    (1..12).each do |i|
-      ground_temps.setTemperatureByMonth(i,10.0)
-    end
+    # disabled I don't see any mention of setting ground temps to 10C
+    #ground_temps = model.getSiteGroundTemperatureBuildingSurface
+    #(1..12).each do |i|
+    #  ground_temps.setTemperatureByMonth(i,10.0)
+    #end
```

Comment:
5.2.4.1.1.2 Soil and Slab Thermal Properties and Boundary Conditions
      a. The soil and floor slab properties shall be constant and equal to the following:
        1. Thermal conductivity = 1.9 W/(m·K) (1.098 Btu/[h·ft·°F]).
        2. Density = 1490 kg/m3 (93.125 lb/ft3).
        3. Specific heat = 1800 J/(kg·K) (0.4302 Btu/[lb°F]).
      b. Initially (time < 0) the ground and slab shall be 10°C (50°F) throughout. (Informative
    Note: At the beginning of the simulation (time = 0), the zone air temperature steps to 30°C
(86°F) per Section 5.2.4.1.1.5, and slab and soil temperatures begin to change accordingly.)